### ISSUE TITLE
File "9.png" missing

### ISSUE THREAD

**Issue by 2nq** [michal-sternik/ggScript#1]
Hello, the image "9.png" is missing from the files, can you please add it? Thanks in advance,

**Comment by michal-sternik** [comment]
@2nq yes, that's true. I've tried to run the code, but it seems that developers of pyautogui library changed the behaviour of "locateOnScreen" function, which now throws an exception if the image is not found (earlier it was just assigning "None" to the image variable). So i would have to completely fix the code, and i do not remember what was on "9.png" (it was named so bad..). For now i do not plan to do it, but it keeps comming back to my head (so i think ill take care of it in the future). Feel free to use the code if you want to fix it and ask if something is unclear.

**Comment by michal-sternik** [comment]
@2nq recently managed to resolve the issue, now the program is ready to use
